faster_whisper/transcribe.py

```diff
@@ -1449,6 +1602,32 @@ def key_func(language):
         # Language is not detected for any segment and none of prev conditions met
         return {'language_code': 'silence','language_confidence': 1.0} 
 
+default_batched_asr_options =  {
```

Comment:
Wouldn't it be better to take it from the function's defaults like it's done in the official faster-whisper? https://github.com/SYSTRAN/faster-whisper/blob/1eb9a8004c509a4af2960955374520c35b7b793a/faster_whisper/transcribe.py#L184

Reply:
There will be a slightly lesser number of parameters for the batched version as well and then pass through the iterator to the forward function. Then it should support all latest additional parameters of the transcribe function but just do not allow static values for batched whisper (word timestamps as a parameter for example). Whats your take on it?

Reply:
Updated the transcribe function and modified pipeline to allow run time parameters.